### ISSUE TITLE
eeg-fmri denoising

### ISSUE THREAD

**Issue by russellu** [mne-tools/mne-python#5704]
hello, i would like to contribute a eeg-fmri denoising pipeline (removing mri artifacts from eeg data acquired during simultaneous eeg-fmri experiments).

i will be developing this in a separate repository on my own github, does anyone know the procedure for merging it with mne once it is complete? thanks,

Russell

**Comment by sappelhoff** [comment]
closing in favor of #10466, which I think aims to do something similar.
